doc/lightning-listchannels.7.txt

```diff
@@ -0,0 +1,65 @@
+LIGHTNING-LISTCHANNELS(7)
+=======================
+:doctype: manpage
+
+NAME
+----
+lightning-listchannels - Command for returning data on active lightning
+channels..
```

Comment:
nit: double `.` at the end.

Reply:
I also prefer "Command to query active lightning channels in the entire network." since it is for all known, not just our own channels.